scripts/jira-automation.py

```diff
@@ -31,6 +33,86 @@ def get_organization(self, org_id: str) -> {}:
         """
         return self.__client.organizations.get(org_id)
 
+    def get_code_analysis_results(
+        self, project_id: str, snyk_org_id: str, snyk_api_token: str
+    ):
+        results = []
+        headers = {
+            "authorization": f"token {snyk_api_token}",
+            "accept": "application/vnd.api+json",
+        }
+
+        params = {
+            "version": "2024-01-23",
+            "limit": "100",
```

Comment:
I guess these 2 should be configurable.

Reply:
you mean, we can set it as env vars and then pass it right?

Reply:
here when you mean 2 , you mean the `version` and `limit right` ?

Reply:
> here when you mean 2 , you mean the `version` and `limit right` ?

yes, 


> you mean, we can set it as env vars and then pass it right?
not as env vars, use the argparse library.

Reply:
made the changes now.. let me know how it looks :)